### PR TITLE
feat: test completeness score

### PR DESCRIPTION
**Pull request by JohnShiuMK**
part of the #39

### CODE REVIEW

**Comment by SoloSynth1** [comment]
If I understand correctly in the latest commit you just changed the code in notebook, and I need to update the python file to reflect that, is that right?

**Comment by tonyshumlh** [comment]
@SoloSynth1 you are correct

File: src/test_creation/04_refactor-code.ipynb [hunk 1]
```diff
@@ -250,21 +252,28 @@
     "            }]\n",
     "\n",
     "        self.evaluation_result = result\n",
-    "        return result\n",
+    "        return\n",
     "\n",
     "    def get_completeness_score(self):\n",
-    "        pass"
+    "        report_df = pd.DataFrame(self.evaluation_result)['report'].explode('report').apply(pd.Series)\n",
```
Comment (SoloSynth1) [inline src/test_creation/04_refactor-code.ipynb:258]: I see you are using `pandas`, but I don't think the current dependency list contains that. I think you can either add it back to `environment.yml`, or come up with a way to make it not dependent on `pandas`.

**Review by SoloSynth1 — COMMENTED** [review]
@tonyshumlh dependency problem. Do you want to add `pandas` as dependency?

File: src/test_creation/04_refactor-code.ipynb (hunk 1 above)
Comment (SoloSynth1) [reply src/test_creation/04_refactor-code.ipynb:258]: added `pandas` as dependency.

**Review by SoloSynth1 — APPROVED** [review]
LGTM! Admittedly we can refactor this, but I think this will work as a MVP for now.

Thanks @JohnShiuMK and @tonyshumlh

**Pull request merged**